modules/installation-guide/partials/proc_importing-untrusted-tls-certificates-old.adoc

```diff
@@ -25,7 +25,7 @@ endif::[]
 ifeval::["{project-context}" != "che"]
 When
 endif::[]
-the cluster contains cluster-wide trusted CA certificates added through the link:https://docs.openshift.com/container-platform/4.4/networking/configuring-a-custom-pki.html#nw-proxy-configure-object_configuring-a-custom-pki[cluster-wide-proxy configuration], {prod-short} Operator detects them and automatically injects them into this ConfigMap:
+the cluster contains cluster-wide trusted CA certificates added using the link:https://docs.openshift.com/container-platform/4.4/networking/configuring-a-custom-pki.html#nw-proxy-configure-object_configuring-a-custom-pki[cluster-wide-proxy configuration], {prod-short} Operator detects them and automatically injects them into this ConfigMap:
```

Comment:
Capital T. "The cluster-wide-proxy configuration" full-stop. The {prod-short} Operator detects and automatically injects them into the ConfigMap".
I somehow don't see why there is a : here.